tlcc_enclave/test/test.c

```diff
@@ -8,6 +8,8 @@
 #include <stdio.h>
 #include <string.h>
 
+#include <glob.h>
```

Comment:
I don't see this `glob.h`

Reply:
this is in `/usr/include/glob.h`.  and should be standard glibc.  On four different systems (including on some wheree i've never installed pdo and/or fpc) it is part of libc6-dev 
```
msteiner@ada2 ~$ dpkg-query -S /usr/include/glob.h
libc6-dev:amd64: /usr/include/glob.h
```